src/opts.rs

```diff
@@ -6,6 +6,8 @@ use clap::{Parser, ValueHint};
 /// A command-line utility for easily compressing and decompressing files and directories.
 ///
 /// Supported formats: tar, zip, bz/bz2, gz, lz4, xz/lz/lzma, zst
```

Comment:
```suggestion
/// Supported formats: tar, zip, bz/bz2, gz, lz4, xz/lz/lzma, zst.
```

Reply:
Commited this period but not in the URL because it breaks how some terminals deal with URL links